app/main.py

```diff
@@ -2,5 +2,14 @@
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    memory = {}
+
+    def inner(*args, **kwargs) -> int:
+        array = tuple([*args])
+        if array in memory.keys():
```

Comment:
Using `if array in memory:` is sufficient and more idiomatic in Python. The `.keys()` method is not needed when checking for the presence of a key in a dictionary.